src/app/view/components/_common/dot-empty-state/dot-empty-state.component.scss

```diff
@@ -0,0 +1,97 @@
+@use "variables" as *;
+
+:host {
```

Comment:
This whole file needs format as my previous comment. Some clases have no break lines.

The clasess don't need to be nested inside `:host`